### DISCUSSION TITLE
Is there a way to define presets for "Customize Columns" in folder comparison screen?

### DISCUSSION THREAD

**Discussion by TFWol** [WinMerge/winmerge#1880] [Q&A] [answered]
Since there are so many columns (Additional Properties) to choose from, I'd like to have custom presets to easily jump between. 
Being able to save the order and sizes of each in the preset would be a bonus. 

When looking at directories, I have to toggle columns on the fly pretty often to speed things up. MD5 is one example.

Am I missing a setting somewhere to define this?

**Comment by sdottaka** [comment, marked as answer]
Unfortunately WinMerge doesn't have such a feature.
However, if you specify a command line argument like `/inifile %USERPROFILE%\wm.ini`, you can read and save configuration information from the specified INI file instead of the registry.
If necessary, switch to his INI file with Additional Properties added.

**Reply by TFWol** [reply]
Ah, too bad. Thanks for the workaround though.
